R/human_infection.R

```diff
@@ -183,12 +179,18 @@ calculate_infections <- function(
   }
 
   prob <- b * (1 - prophylaxis) * (1 - vaccine_efficacy)
-  infected <- bitset_at(source_humans, bernoulli_multi_p(prob))
 
+  ## All bitten humans with an infectious bite (incorporating prophylaxis and vaccines)
+  infected <- bitset_at(bitten_humans, bernoulli_multi_p(prob))
```

Comment:
We should use `bernoulli` here if we're simulating vivax and the probability of infection is constant because it is much faster.

Reply:
While the probability of infection (b) is constant, it can still be affected by prophylaxis and prospective vaccines (such that prob is not constant), so bernoulli_multi_p is still appropriate.